docs/Process Decisions

```diff
@@ -0,0 +1,64 @@
+Design & Development Process Decisions
```

Comment:
This wasn't the document in question -> we needed the other document we wrote earlier.
Also this isnt markdown formatted -> we should have this file in the repo but it needs to be in markdown